src/applications/hca/tests/00-all-fields.e2e.spec.js

```diff
@@ -42,6 +73,27 @@ module.exports = E2eHelpers.createE2eTest(client => {
   // Birth information page.
   client.expect.element('select[name="root_veteranDateOfBirthMonth"]').to.be
     .visible;
+  if (process.env.BUILDTYPE !== ENVIRONMENTS.VAGOVPROD) {
+    // check that id-form values have been copied.
+    const [
+      vetDobYr,
+      vetDobMo,
+      vetDobDy,
```

Comment:
I'd really prefer to spell out year, month, and day here. I know it's just my opinion but I really don't like using abbreviations when naming things unless they are really common abbreviations. Code completion is our friend so abbreviations often don't save a lot of typing and just have the potential to introduce confusion. What do you think, @ncksllvn ?

Reply:
Done.